src/wallet/wallet.h

```diff
@@ -365,82 +365,24 @@ struct COutputEntry
     int vout;
 };
 
-/** A transaction with a merkle branch linking it to the block chain. */
+/** Legacy class used for deserializing vtxPrev for backwards compatibility **/
```

Comment:
Could add the commit or wallet version where this was changed?

Reply:
I've expanded the comment here